app/models/payment.js

```diff
@@ -40,4 +40,4 @@ class Payment {
   }
 }
 
-module.exports = Payment
+module.exports = {Payment}
```

Comment:
what need for this as opposed to just exporting the class directly?

Reply:
The rational is `models/product.js` now exports an object, so just for the sake of uniformity I made this one return an object as well.